Argument error: function expected 2 arguments, but was called with 1 arguments. Error occured in call:
foo(42)